test/integration/targets/pause/runme.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eux
+
+export ANSIBLE_ROLES_PATH=./roles
+
+python runme.py -i ../../inventory "$@"
```

Comment:
You shouldn't need a `runme.sh` just to run `runme.py`, since ansible-test will execute `runme.py` directly.

Reply:
However, to avoid having to install `pexpect` for every integration test run via `integration.txt`, it may be better to keep this for now and install `pexepect` via pip here rather than using `integration.txt`.

Reply:
Thanks for answering that! I was going to ask you about that today but you beat me to it.